charts/cp-schema-registry/templates/deployment.yaml

```diff
@@ -20,10 +20,14 @@ spec:
   template:
     metadata:
       labels:
+      {{- if .Values.datadog.enabled }}
+        {{- include "datadog.labels" $ | indent 8 }}
+      {{- end }}
```

Comment:
This will add the labels on the **pod**, and it's good 👍 

I think we **also** need to add the same include on the **Deployement** itself, so a few lines above.
I don't remember exactly why technically, but that's the "integration method" that we validated a few months ago and applied for other technos (cassandra, mariadb for example).

Reply:
Did we write the decision log about this somewhere ? 
In this SRE doc about unified tagging I can see the labels but I can't see the kube resource on which we need to apply them, maybe we need to edit it 
 if it's useful to add them on the meta resource (sts or deployment )https://blablacar.atlassian.net/wiki/spaces/CD/pages/1361248359/Datadog+Unified+Tagging

Also, in the Datadog docs for out of the box tags https://docs.datadoghq.com/agent/kubernetes/tag/?tab=containerizedagent#out-of-the-box-tags, I can see that they specify to put them in the pods but I. can't see something on the meta resources

Reply:
I can see only `Pod label must exist`

Reply:
I can see here that there is an example for which it is useful indeed https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes

I'm going to add them as well

Reply:
Maybe we should put it in a decision record somewhere so that we know why exactly we do it.

Reply:
> Did we write the decision log about this somewhere ?
I don't really know. If you already searched and didn't found it, maybe not.
@aymeric-ledizes might know better than me.

Reply:
There is no write decision on this but we just decided to follow the Datadog doc as you shared above https://docs.datadoghq.com/getting_started/tagging/unified_service_tagging/?tab=kubernetes#full-configuration

> To get the full range of unified service tagging when using Kubernetes, add environment variables to both the deployment object level and the pod template spec level: